frontend/lib/generate-image-markup.ts

```diff
@@ -1,8 +1,8 @@
-export function bareLink(badgeUrl, link, title = '') {
+export function bareLink(badgeUrl: string, link?: string, title = '') {
```

Comment:
Do we really still need `link` and `title` parameters? Doesn't look like they were ever used.

Reply:
It gives these functions a common signature, which I think is helpful.

Reply:
Ah so consistency for the callers

Reply:
Right, and now that types are checked it makes that `generateMarkup` function work.